src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,24 @@
+import { Errors } from '../types/Errors';
+import { FilterBy } from '../types/FiilterBy';
+import { Todo } from '../types/Todo';
+
+type FilterTheTodos = (todos: Todo[], filterBy: FilterBy) => Todo[];
+
+export const getFilteredTodos: FilterTheTodos = (todos, filterBy) => {
+  let filteredTodos = todos;
```

Comment:
You don't need let in this case. You can use const and then return action with the array.